Sources/Authentication/Persist/RedirectMiddleware.swift

```diff
@@ -0,0 +1,17 @@
+import Vapor
+
+public struct RedirectMiddleware<A>: Middleware where A: Authenticatable {
+
+    let path: String
+
+    public init(A authenticableType: A.Type = A.self, path: String) {
```

Comment:
Static extension on Authenticatable to match the other middleware would be nice.

```swift
router.grouped(User.guard(redirectingTo: "/login")).get(...) { ... }
```